core/store/src/db/refcount.rs

```diff
@@ -1,9 +1,22 @@
+//! Functions for handling reference counted columns.
+//!
+//! Some of the columns in the database are reference counted.  Those are the
+//! ones for which [`crate::DBCol::is_rc`] returns `true`.  Inserting value to
+//! such column increases the reference count and removing value decreases it.
+//! The key is really removed from the database when reference count reaches
+//! zero.
```

Comment:
Should we also document what happens if different values are inserted for the same key?